dev-python/pytest-twisted/pytest-twisted-1.13.4.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit distutils-r1
+
+DESCRIPTION="Test twisted code with pytest"
+HOMEPAGE="https://github.com/pytest-dev/pytest-twisted"
+SRC_URI="https://github.com/pytest-dev/pytest-twisted/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	dev-python/decorator[${PYTHON_USEDEP}]
+	dev-python/greenlet[${PYTHON_USEDEP}]
+	>=dev-python/pytest-2.3[${PYTHON_USEDEP}]
+	dev-python/twisted[${PYTHON_USEDEP}]
+"
+DEPEND=""
+BDEPEND=""
+
+distutils_enable_tests pytest
+
+src_prepare() {
+	# https://github.com/pytest-dev/pytest-twisted/issues/146
+	use python_targets_python3_10 &&
+		eapply "${FILESDIR}/${P}-py310-ignore-deprecated-currentThread.patch"
```

Comment:
Can we apply this unconditionally?

Reply:
Actually, looks like it was working around a bug in older versions of dev-python/twisted itself, which are no longer in ::gentoo. Package passes tests without the patch on 3.10 && 3.11 so I'm calling it fixed!